### PR TITLE
Fix some types

### PR DESCRIPTION
**Pull request by sbrunner**
Add option noImplicitAny
Use types from definitively typed

### CODE REVIEW

**Review by ger-benjamin — APPROVED** [review]
Quick look, looks good.
But that's too long for a human I let typecheck give you the GO.

File: contribs/gmf/src/datasource/ExternalDataSourcesManager.js [hunk 1]
```diff
@@ -336,8 +337,8 @@ export class ExternalDatSourcesManager {
   }
 
   /**
-   * @param {Object} layer WMS Capability Layer object.
-   * @param {Object} capabilities  WMS Capabilities definition
+   * @param {Object<any, any>} layer WMS Capability Layer object.
```
Comment (ger-benjamin) [inline contribs/gmf/src/datasource/ExternalDataSourcesManager.js:340]: Is there no better definition ? `Object{string, any}` ?

Comment (sbrunner) [reply contribs/gmf/src/datasource/ExternalDataSourcesManager.js:340]: Good catch, thanks :-)

**Comment by sbrunner** [comment]
I agree, Thanks @ger-benjamin :-)

**Comment by sbrunner** [comment]
Finally, I didn't use @type/ol, but I extract the typing definitions from the OpenLayers files.

File: Makefile [hunk 2]
```diff
@@ -360,6 +360,13 @@ contribs/dist: .build/build-dll.timestamp
 .build/node_modules.timestamp: package.json
 	npm install
 	mkdir -p $(dir $@)
+	(cd node_modules/ol; curl https://raw.githubusercontent.com/ahocevar/openlayers/main/config/tsconfig-build.json > tsconfig.json)
```
Comment (sbrunner) [inline Makefile:363]: ahocevar => master

**Pull request merged**